modules/overview-of-vector-databases.adoc

```diff
@@ -1,26 +1,33 @@
 
-Vector databases provide you with the following capabilities: 
+Vector databases provide the following capabilities:
 
 * Store vector embeddings generated by embedding models.
 * Support efficient similarity search to retrieve semantically related content.
 * Enable RAG workflows by supplying the LLM with contextually relevant data from a specific domain.
 
-When you deploy RAG workloads in {productname-short}, you can deploy vector databases through the Llama Stack Operator. Currently, {productname-short} supports the following vector databases:
+When you deploy RAG workloads in {productname-short}, you can deploy vector databases through the Llama Stack Operator.  
+Currently, {productname-short} supports the following vector databases:
 
 * *Inline Milvus Lite*  
-An Inline Milvus vector database runs embedded within the Llama Stack Distribution (LSD) pod and is suitable for lightweight experimentation and small-scale development. Inline Milvus stores data in a local SQLite database and is limited in scale and persistence.
+  An Inline Milvus vector database runs embedded within the Llama Stack Distribution (LSD) pod and is suitable for lightweight experimentation and small-scale development. Inline Milvus stores data in a local SQLite database and is limited in scale and persistence.
+
+* *Inline FAISS*  
+  Inline FAISS provides an alternative lightweight vector store for RAG workloads. FAISS (Facebook AI Similarity Search) is an open-source library for efficient similarity search and clustering of dense vectors. When configured inline with a SQLite backend, FAISS runs entirely within the Llama Stack container and stores embeddings locally without requiring a separate database service. Inline FAISS is ideal for testing and experimental RAG deployments.
 
 * *Remote Milvus*  
-A remote Milvus vector database runs as a standalone service in your project namespace or as an external managed deployment. Remote Milvus is recommended for production-grade RAG use cases because it provides persistence, scalability, and isolation from the Llama Stack Distribution (LSD) pod. In {openshift-platform} environments, you must deploy Milvus with an etcd service directly in your project. For more information on using etcd services, see https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp-latest-version}/html/etcd/index[Providing redundancy with etcd]. 
+  A remote Milvus vector database runs as a standalone service in your project namespace or as an external managed deployment.  
+  Remote Milvus is recommended for production-grade RAG use cases because it provides persistence, scalability, and isolation from the Llama Stack Distribution (LSD) pod.  
```

Comment:
```suggestion
  Remote Milvus is best for production-grade RAG use cases because it provides persistence, scalability, and isolation from the Llama Stack Distribution (LSD) pod.  
```